Frends.Opcua.Write/Frends.Opcua.Write.Tests/generate-certs.sh

```
@@ -0,0 +1,151 @@
+mkdir -p "$OUTPUT_DIR"
+
+echo "==> Setting permissions on PKI directory..."
+chmod -R 777 "$PKI_DIR"
+
+echo "==> Copying DER to trusted-user store: $TRUSTED_USER_DIR"
+cp "$WORK_DIR/user.der" "$TRUSTED_USER_DIR/user.der"
+
+echo "==> Copying PFX to output directory: $OUTPUT_DIR"
+cp "$WORK_DIR/user.pfx" "$OUTPUT_DIR/user.pfx"
+
+cat > $OUTPUT_DIR/nodesfile.json << 'EOF'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)